### PR TITLE
LA-1864 - Customize Orval package

### PR DESCRIPTION
**Pull request by darshan09200**
## Summary

This PR refactors the SDK from `@orval` to `@vetster/orval`

## Changes

- Renamed `@orval/*` to `@vetster/orval-*`
- Updated dependencies to use new package names
- Updated imports to use new package name
- Updated `release-it` configuration
- Updated author and repo information for packages
- Updated scripts for `tests`
- Updated samples
- Updated prettier config to ignore `*.lock` files
- Updated `.npmrc` config file to point to vetster registry
- Updated release script to use `@vetster/orval` script

### CODE REVIEW

File: tests/package.json [hunk 1]
```diff
@@ -5,24 +5,23 @@
...
-    "generate:default": "yarn orval --config ./configs/default.config.ts",
-    "generate:axios": "yarn orval --config ./configs/axios.config.ts",
-    "generate:angular": "yarn orval --config ./configs/angular.config.ts",
-    "generate:react-query": "yarn orval --config ./configs/react-query.config.ts",
-    "generate:svelte-query": "yarn orval --config ./configs/svelte-query.config.ts",
-    "generate:vue-query": "yarn orval --config ./configs/vue-query.config.ts",
-    "generate:swr": "yarn orval --config ./configs/swr.config.ts",
-    "generate:multi-file": "yarn orval --config ./configs/multi-file.config.ts",
-    "generate:zod": "yarn orval --config ./configs/zod.config.ts",
+    "@vetster/orval": "node ../packages/orval/dist/bin/orval.js",
```
Comment (tw99) [inline tests/package.json]: Do we want to rename the scripts? Seems unnecessary to me

Comment (darshan09200) [reply tests/package.json]: I replaced every `orval` with `@vetster/orval`. I can revert the script name if that works.

File: packages/orval/package.json [hunk 2]
```diff
@@ -33,14 +37,6 @@
     "zod",
     "hono"
   ],
-  "author": {
-    "name": "Victor Bury",
-    "email": "victor@anymaniax.com"
-  },
```
Comment (tw99) [inline packages/orval/package.json:39]: ed2

Comment (darshan09200) [reply packages/orval/package.json:39]: Should I retain the author details?

Comment (tw99) [reply packages/orval/package.json:39]: Should be fine as is

**Pull request merged**